Sources/Logging/Strings/ConfigureStrings.swift

```diff
@@ -187,6 +189,18 @@ extension ConfigureStrings: LogMessage {
             return "Purchases is not configured with StoreKit 2 enabled. This is required in order to detect " +
             "transactions coming from SwiftUI paywalls. You must use `.with(storeKitVersion: .storeKit2)` " +
             "when configuring the SDK."
+
+        case .observer_mode_required:
```

Comment:
This is specific to handling transactions, maybe:
```suggestion
        case .handle_transaction_observer_mode_required:
```